cli/commands/edit_test.go

```diff
@@ -35,10 +35,19 @@ func TestEditCommandLaunchesEditor(t *testing.T) {
 	defer os.Setenv("EDITOR", os.Getenv("EDITOR"))
 	os.Setenv("EDITOR", arbitraryEditor)
 
-	command := &editCommand{
+	initCommand := &initCommand{
```

Comment:
I think you should make a new file to test the `init` stuff

Reply:
We should do that. In this case,  we have to call `init` on the file first, or `edit` will return an error saying that the file isn't tracked. It would be good to make a helper function to do this.